src/interface/web/app/settings/page.tsx

```diff
@@ -0,0 +1,999 @@
+                                            <WhatsappLogo className="h-7 w-7 mr-2"/>
+                                            Chat on Whatsapp
+                                            {numberValidationState === PhoneNumberValidationState.Verified && (
+                                                <CheckCircle weight="bold" className="h-4 w-4 ml-1 text-green-400" />
+                                            ) || numberValidationState !== PhoneNumberValidationState.Setup && (
+                                                <ExclamationMark weight="bold" className="h-4 w-4 ml-1 text-yellow-400" />
+                                            )}
+                                        </CardHeader>
+                                        <CardContent className="grid gap-4">
+                                            <p className="text-gray-400">
+                                                Connect your number to chat with Khoj on WhatsApp. Learn more about the integration <a href="https://docs.khoj.dev/clients/whatsapp">here</a>.
+                                            </p>
+                                            <div>
+                                            {numberValidationState === PhoneNumberValidationState.VerifyOTP && (
+                                                <>
+                                                    <p>{`Enter the OTP sent to your WhatsApp number: ${number}`}</p>
+                                                    <InputOTP
+                                                        autoFocus={true}
+                                                        maxLength={6}
+                                                        value={otp || ""}
+                                                        onChange={setOTP}
+                                                        onComplete={() => setNumberValidationState(PhoneNumberValidationState.Verified)}
+                                                    >
+                                                        <InputOTPGroup>
+                                                            <InputOTPSlot index={0} />
+                                                            <InputOTPSlot index={1} />
+                                                            <InputOTPSlot index={2} />
+                                                            <InputOTPSlot index={3} />
+                                                            <InputOTPSlot index={4} />
+                                                            <InputOTPSlot index={5} />
+                                                        </InputOTPGroup>
+                                                    </InputOTP>
+                                                </>
+                                            ) || (
+                                                <>
+                                                    <Input
```

Comment:
Would be nice to have a country code input drop down. Otherwise, maybe in the placeholder specify that you need a country code, e.g., +18008088000

Reply:
Done